src/components/messenger/chat/styles.scss

```diff
@@ -157,6 +157,14 @@ $recent-indicator-size: 8px;
     color: theme.$color-greyscale-11;
   }
 
+  &__icon-collapse {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 32px;
+    height: 32px;
```

Comment:
i little confused here. We pass `size{..}` into the `IconButton` in zUI and we're also setting the height & width here? What's the difference b/w the two?

Reply:
```
  render() {
    return (
      <button className={classNames('icon-button', this.props.className)} onClick={this.handleClick}>
        <this.props.Icon label={this.props.label} size={this.props.size} isFilled={this.props.isFilled} />
      </button>
    );
  }
  ```
  
You can see in `IconButton` here there is the button container wrapped around the icon. We need to pass this a className and set the height/width relevant to the size passed to the Icon itself. So for 24px size passed to the icon, we want to set the container (button) at height and width of 32px so the background/on hover aligns. You can see this in the figma design here:

The icon size 24px:
<img width="251" alt="Screenshot 2023-06-27 at 10 24 14" src="https://github.com/zer0-os/zOS/assets/39112648/e939b4ea-cb6f-428a-a375-8bedb646059b">

The button container 32px as stated bottom right of screenshot.
<img width="792" alt="Screenshot 2023-06-27 at 10 24 25" src="https://github.com/zer0-os/zOS/assets/39112648/8427b8db-db38-4dab-9ee2-49bf4b2c8f72">

Reply:
ahhhhh, so the container is 32px, but the "icon size" is actually 24px. I didn't knew that before, thought you just need to take the icon size